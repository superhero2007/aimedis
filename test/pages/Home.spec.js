import React from 'react';
import { shallow } from 'enzyme';

import Home from 'pages/Home';

function setup() {
  return shallow(<Home />);
}

describe('Home', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
