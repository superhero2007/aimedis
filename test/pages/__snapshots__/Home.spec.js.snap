// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render properly 1`] = `"<div class=\\"app__home app__route\\"><div class=\\"container\\"><section class=\\"app__home__section1\\"><div class=\\"app__home__section1__content\\"><div class=\\"row\\"><div class=\\"col-lg-6 col-md-12 p-lg-0\\"><div class=\\"app__home__section1__content__header\\">Aimedis is the health platform of the future.</div><div class=\\"app__home__section1__content__subheader d-none d-lg-block\\">Aimedis has been developed by doctors for patients, doctors and hospitals - designed to give patients the complete control over their health data and support.</div><div class=\\"app__home__section1__content__image d-block d-lg-none\\"><div class=\\"app__video__mask\\"><div class=\\"app__video__mask__circle app__video__mask__circle1\\"></div><div class=\\"app__video__mask__circle app__video__mask__circle2\\"></div><div class=\\"app__video__mask__circle app__video__mask__circle3\\"></div><div class=\\"app__video__mask__triangle\\"></div></div></div><div class=\\"app__home__section1__content__get__access\\"><div class=\\"app__get__early__access\\"><input type=\\"text\\" class=\\"app__get__early__access__input\\" placeholder=\\"Your Email Address\\"/><button class=\\"app__get__early__access__button btn btn__primary\\">Get <span class=\\"d-none d-lg-inline\\">Early </span>Access</button></div></div><div class=\\"app__home__section1__content__terms\\">By filling my email and clicking on “GET EARLY ACCESS“ I agree to the <span>Conditions Of Use</span> and the <span>Data Protection Policy</span>.</div></div><div class=\\"col-lg-5 offset-lg-1 d-none d-lg-block app__home__section1__image\\"><div class=\\"app__video__mask\\"><div class=\\"app__video__mask__circle app__video__mask__circle1\\"></div><div class=\\"app__video__mask__circle app__video__mask__circle2\\"></div><div class=\\"app__video__mask__circle app__video__mask__circle3\\"></div><div class=\\"app__video__mask__triangle\\"></div></div><div class=\\"app__home__mask__one\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div></div></section><section class=\\"app__home__section2\\"><div class=\\"app__home__section2__content\\"><div class=\\"row\\"><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div><div class=\\"col-lg-2 col-6 app__home__section2__content__item\\"><div class=\\"app__avatar\\"><img class=\\"app__avatar__image\\" src=\\"\\" alt=\\"No Image\\"/></div></div></div></div></section><section class=\\"app__home__section3\\"><div class=\\"app__home__section3__title text-center\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Platform features</p><p class=\\"app__title__title\\">What Aimedis offers</p></div></div><div class=\\"app__home__section3__content\\"><div class=\\"row\\"><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Your health data in one place</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Videochat with doctors</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item active\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Doctors appointments</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 offset-lg-3 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Independent Second Opinion</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Connect fitness trackers</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Care for your family</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 offset-lg-3 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Prescription online</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Medical tourism</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"col-lg-3 col-6 app__home__section3__content__item\\"><div class=\\"app__item\\"><div class=\\"app__item__content\\"><div class=\\"app__item__content__icon mx-auto\\"><img class=\\"app__item__content__icon__img\\" src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__item__content__icon__active\\"><div class=\\"app__item__content__icon__active__circle\\"></div><div class=\\"app__item__content__icon__active__triangle\\"></div></div></div><p class=\\"app__item__content__title mx-auto\\">Your health data in one place</p><p class=\\"app__item__content__active mx-auto\\">See full product overview</p></div><div class=\\"app__item__background\\">Image<img src=\\"\\" alt=\\"No Image\\"/></div></div></div></div></div></section><section class=\\"app__home__section4\\"><div class=\\"app__home__section4__content\\"><div class=\\"row\\"><div class=\\"col-lg-5 col-md-12\\"><div class=\\"text-center text-lg-left mx-auto ml-lg-0\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">forward thinking</p><p class=\\"app__title__title\\">The Technology</p></div></div><p class=\\"app__home__section4__content__subtitle\\">Blockchain smart contracts smart graphs artificial intelligence highest security standards payments cryptocurrencies ICD ATC HL7 FHIR HIPPA IHE DICOM.</p><p class=\\"app__home__section4__content__subtitle\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help patients to support each other on a new level.</p><button class=\\"app__home__section4__content__download body btn btn__secondary ml-0\\" type=\\"button\\">DOWNLOAD TECH SPEC (3.4 MB)</button></div><div class=\\"col-lg-7 d-none d-lg-block\\"><div class=\\"row\\"><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">Optimal Security</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">Blockchain &amp; Smart contracts</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">Smart graphs &amp; AI</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">Payment &amp; cryptocurrencies</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">AIMSafe drug safety</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">ICD 10</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">HL7, FHIR, DICOM, HIPAA, IHE</p></div></div><div class=\\"col-lg-6 app__home__section4__content__item\\"><div class=\\"app__list\\"><div class=\\"app__list__icon\\"><img class=\\"app__list__icon__img\\" src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__list__title\\">ATC</p></div></div></div></div></div></div></section><section class=\\"app__home__section5\\"><div class=\\"app__home__section5__title text-center\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Platform Overview</p><p class=\\"app__title__title\\">Aimedis Ecosystem</p></div></div><div class=\\"app__home__section5__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-5\\"><div class=\\"app__overview active\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">1</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Fully responsive</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">In every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">2</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Multilanguage</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">also fully functional abroad</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">3</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Easy to use</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">also for unexperienced patients simple &amp; fast</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">4</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">ISO 21007 certification</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">For optimal security through regular security audits</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">5</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Tested by doctors &amp; hospitals</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">Tested by active physicians and medical IT specialists</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div></div><div class=\\"col-lg-7 d-none d-lg-block\\"><img src=\\"\\" alt=\\"No Image\\"/><div class=\\"app__home__mask__four\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div></div></section><section class=\\"app__home__section6\\"><div class=\\"app__home__section6__title text-center\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">what users say</p><p class=\\"app__title__title\\">Testimonials</p></div></div><div class=\\"app__home__section6__content\\"><div class=\\"app__carousel\\"><div class=\\"app__carousel__content\\"><div class=\\"app__carousel__content__inner\\"><div class=\\"app__carousel__content__inner__item prev\\"><div class=\\"app__carousel__content__indicator__prev\\"><span class=\\"i-long-arrow-left\\"></span></div><div class=\\"app__carousel__content__inner__item__content\\"><img src=\\"\\" alt=\\"Carousel Image\\"/><p class=\\"app__carousel__content__inner__item__description\\">&quot;Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick, easy and efficient treatment.&quot;</p><p class=\\"app__carousel__content__inner__item__name\\">Rizmy Shariff</p><p class=\\"app__carousel__content__inner__item__caption\\">Family doctor Solingen Germany</p><p class=\\"app__carousel__content__inner__item__quotation\\">“</p></div></div><div class=\\"app__carousel__content__inner__item active\\"><div class=\\"app__carousel__content__inner__item__content\\"><img src=\\"\\" alt=\\"Carousel Image\\"/><p class=\\"app__carousel__content__inner__item__description\\">&quot;Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick, easy and efficient treatment.&quot;</p><p class=\\"app__carousel__content__inner__item__name\\">Rizmy Shariff</p><p class=\\"app__carousel__content__inner__item__caption\\">Family doctor Solingen Germany</p><p class=\\"app__carousel__content__inner__item__quotation\\">“</p></div></div><div class=\\"app__carousel__content__inner__item next\\"><div class=\\"app__carousel__content__inner__item__content\\"><img src=\\"\\" alt=\\"Carousel Image\\"/><p class=\\"app__carousel__content__inner__item__description\\">&quot;Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick, easy and efficient treatment.&quot;</p><p class=\\"app__carousel__content__inner__item__name\\">Rizmy Shariff</p><p class=\\"app__carousel__content__inner__item__caption\\">Family doctor Solingen Germany</p><p class=\\"app__carousel__content__inner__item__quotation\\">“</p></div><div class=\\"app__carousel__content__indicator__next\\"><span class=\\"i-long-arrow-right\\"></span></div></div></div></div></div></div></section><section class=\\"app__home__section7 d-none d-lg-block\\"><div class=\\"app__home__section7__title\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">information on the go</p><p class=\\"app__title__title\\">Aimedis is mobile</p></div></div><div class=\\"app__home__section7__content\\"><div class=\\"app__home__section7__content__mobile app__home__section7__content__iphone\\"><div class=\\"app__home__section7__content__mobile__img app__home__section7__content__iphone__img mx-auto\\"><img src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__home__section7__content__mobile__title app__home__section7__content__iphone__title\\">on iPhone and iPad from iOS 9</p><button class=\\"btn btn__secondary app__home__section7__content__mobile__button app__home__section7__content__iphone__button\\">GO TO App Store</button></div><div class=\\"app__home__section7__content__mobile app__home__section7__content__android\\"><div class=\\"app__home__section7__content__mobile__img app__home__section7__content__android__img mx-auto\\"><img src=\\"\\" alt=\\"No Image\\"/></div><p class=\\"app__home__section7__content__mobile__title app__home__section7__content__android__title\\">on Android smartphones and tablet from Android 4</p><button class=\\"app__home__section7__content__mobile__button btn btn__primary app__home__section7__content__android__button\\">GO TO GOOGLE PLAY</button><div class=\\"app__home__section7__content__mobile__soon\\">soon</div></div><div class=\\"app__home__section7__content__img ml-auto\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></section><section class=\\"app__home__section8\\"><div class=\\"app__home__section8__title text-center\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Learn what’s new</p><p class=\\"app__title__title\\">Aimedis Blog</p></div></div><div class=\\"app__home__section8__content app__home_section8__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\"/></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/2\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/3\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></a></div></div></div></section><section class=\\"app__home__section9\\"><div class=\\"app__feature\\"><div class=\\"app__feature__title\\">Discover the best features for you.</div><div class=\\"app__feature__content\\"><div class=\\"app__get__early__access\\"><input type=\\"text\\" class=\\"app__get__early__access__input\\" placeholder=\\"Your Email Address\\"/><button class=\\"app__get__early__access__button btn btn__primary\\">Get <span class=\\"d-none d-lg-inline\\">Early </span>Access</button></div></div></div></section><section class=\\"app__home__mask d-none d-lg-block\\"><div class=\\"app__home__mask__two\\"><img src=\\"\\" alt=\\"No Image\\"/></div><div class=\\"app__home__mask__three\\"><img src=\\"\\" alt=\\"No Image\\"/></div><div class=\\"app__home__mask__five\\"><img src=\\"\\" alt=\\"No Image\\"/></div><div class=\\"app__home__mask__notification\\"><div class=\\"app__notification\\"><div class=\\"app__notification__icon\\"><img src=\\"\\" alt=\\"No Image\\"/></div><div class=\\"app__notification__content\\"><div class=\\"app__home__mask__notification__content\\"><div class=\\"app__home__mask__notification__content__title\\"><span>Token sale</span> is currently live!</div><div class=\\"app__home__mask__notification__content__description\\">Ending in <span>12d 15h 3m 44s</span></div></div></div><div class=\\"app__notification__close\\">x<i class=\\"i-times\\"></i></div></div></div></section></div></div>"`;
